### ISSUE TITLE
Accessibility Issues - Preferences

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#8641]
### Bug description

Including multiple accessibility issues into one task for easier. 
1. If the User presses 'Tab' key, the first items that should be selected is the first item of the left side menu. 
2. Tabbing should scroll Preferences tabs that are scrollable
3. Scrollbars should be persistent in Preferences tabs that are too large
4. (Audio Settings) Make Buffer Length and Latency compensation tabbable
5. (Audio Settings) Default Sample Format is not tabbable
6. (Playback/Recording) Short Skip and Long Skip should be scrollable
7. (General) File button should be tabbable under 'Temporary files location'

### Steps to reproduce

1. Launch Audacity 4
2. Open Preferences 
3. Go through the list mentioned above

### Expected behavior

All 7 items should be fixed. 

### Audacity Version

5bfbb6df56731cc8156b740b93cfa2ca9126b223

### Operating system

OS independent

**Comment by DavidBailes** [comment]
> Including multiple accessibility issues into one task for easier.
> 
> 1. If the User presses 'Tab' key, the first items that should be selected is the first item of the left side menu.

I think that when the dialog opens, the initial focus should be the item selected in the list of categories (the column on the left hand side).
 
> 5. (Audio Settings) Make Buffer Length and Latency compensation tabbable
> 6. (Audio Settings) Default Sample Format is not tabbable
> 7. (Playback/Recording) Short Skip and Long Skip should be scrollable
> 8. (General) File button should be tabbable under 'Temporary files location'

Issues 5-8 arise because of the non-standard keyboard navigation in this dialog. My understanding is that the dialog will be changed to use standard keyboard navigation, see: https://github.com/audacity/audacity/issues/7786#issuecomment-2510990699

**Comment by chinakov** [comment]
Separate issue for point 3: https://github.com/audacity/audacity/issues/8783

**Comment by chinakov** [comment]
AU4-only issue.

**Comment by chinakov** [comment]
Still reproducible on the latest Master (190bedb093e93c43c654223364c8d52ac07f5062)

**Comment by chinakov** [comment]
This is fixed on the latest Master (6a2ccc6bae123089f852ee22b47c819118b94211)

**Comment by chinakov** [comment]
These issues are still present:
(Audio Settings) Make Buffer Length and Latency compensation tabbable
(Audio Settings) Default Sample Format is not tabbable

**Comment by DavidBailes** [comment]
See discussion about keyboard navigation in dialog boxes: https://github.com/audacity/audacity/discussions/10256
